src/compositor/lipstickcompositor.cpp

```diff
@@ -732,3 +737,35 @@ void LipstickCompositor::readContent()
 {
     m_recorder->recordFrame(this);
 }
+
+void LipstickCompositor::timerEvent(QTimerEvent *e)
+{
+    if (isVisible()) {
+        stopRendering();
```

Comment:
Not sure I understand why compositor needs to keep drawing, I guess this relates to lock-ups mentioned in the other pr. Other thing is that since hide() is delayed, there is at least a possibility that touch events are received by the window (unless grabbed by mce).